test/Api.Tests/TestStartup.cs

```diff
@@ -14,7 +12,9 @@ public void ConfigureTestServices(IServiceCollection services)
             base.ConfigureServices(services);
 
             // Register here the dependency injection services
-           // services.AddDbContext<ApplicationDbContext>(opts => opts.UseInMemoryDatabase());
```

Comment:
As i told you, you shoud create an own Startup class for tests, if you don't wan't to use PostgreSQL etc.